config/routes.rb

```diff
@@ -10,11 +10,17 @@
 
   patch '/merchants/:id', to: 'merchants#update'
   patch '/merchants/:merchant_id/items/:id', to: 'items#update'
-  
+
   get 'merchants/:id/dashboard', to: 'merchants#show'
+  # patch 'items/:id', to: 'items#update'
+
 
   namespace :admin do
     resources :merchants, except: [:update]
   end
+
+  resources :admin, only: [:index]
```

Comment:
I like that! I have never written something like this before so it had skipped my mind that you could do that!